src/Student6/Ex2/Counter.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react'
+
+export const TenCounter = () => {
+  let [counter, setCounter] = useState(0);
+
+  const onhandleClick = () => {
+    if (counter <10) {setCounter(++counter);} 
```

Comment:
Let's format this better:

if () {
  ...
} else {
....
}

Also the name of this PR is for Ex 3, but we are working on Ex 2.

Reply:
1. Fix if else formatting
2. Change name from Up Down Counter to TenCounter